benchopt/runner.py

```diff
@@ -493,5 +495,8 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
 
     if plot_result:
         from benchopt.plotting import plot_benchmark
-        plot_benchmark(save_file, benchmark)
+        plot_benchmark(save_file, benchmark, html)
+    
+    	
+    	
```

Comment:
Only use 1 blank line between successive line of code, 2 blank line between functions/classes definition.
```suggestion
        plot_benchmark(save_file, benchmark, html)

```

Reply:
Also the error in the test seems to come from here.
Can you look at the `plot_benchmark` function signature? What is the 3rd positional argument?